package.json

```
@@ -6,7 +6,8 @@
     "test:watch": "jest --watchAll",
     "test:update": "jest -u",
     "dev": "next",
-    "lint-check": "eslint ./src ./pages --ext .js --max-warnings 0",
+    "lint": "echo 'No lint step for this project; use lint:manual and lint:fix locally'",
```

Should this be lint-check ?
```suggestion
    "lint": "echo 'No lint step for this project; use lint:check and lint:fix locally'",
```